Cargo.toml

```diff
@@ -131,3 +131,4 @@ serde_json = "1.0.120"
 
 # other crates
 thiserror = "2.0.12"
+zerocopy = "0.8.24"
```

Comment:
As you don't use `= { default-features = false; }` I think you also don't need `{ workspace = true, features = ["derive"] }` at other places.

However, a cleaner approach would be to deactivate all features at the top-level and stick to `{ workspace = true, features = ["derive"] }` where you need it